editor/editor_inspector.cpp

```diff
@@ -3889,10 +3893,9 @@ void EditorInspector::_notification(int p_what) {
 			changing++;
 
 			if (update_tree_pending) {
-				update_tree();
+				// If there is a new update tree request during _update_tree, defer it to the next frame to ensure that the built tree is eventually correct.
 				update_tree_pending = false;
-				pending.clear();
```

Comment:
Why did you remove `pending.clear()`? AFAIK it makes single properties refresh and if whole inspector is refreshed, no point in updating the properties further.

Reply:
Yep, thanks for pointing it out.